_posts/zh/2019-10-02-btse-exchange-operation.md

```diff
@@ -0,0 +1,15 @@
+---
+title: '实地报告：BTSE 使用比特币技术进行交易所运营'
+permalink: /zh/btse-exchange-operation/
+name: 2019-10-02-btse-exchange-operation-zh
+slug: 2019-10-02-btse-exchange-operation-zh
+type: posts
+layout: post
+lang: zh
+version: 1
+
+excerpt: >
+  一份关于 BTSE 如何使用比特币技术改进其交易所运营的实地报告。
+
+---
+{% include articles/dong-btse-operation.md hlevel='##' %}
```

Comment:
Do you plan to translate the body content of the post?

Reply:
Actually looks like you have here: https://deploy-preview-1924--bitcoinops.netlify.app/zh/newsletters/2019/10/02/#%E5%AE%9E%E5%9C%B0%E6%8A%A5%E5%91%8Abtse-%E4%BA%A4%E6%98%93%E6%89%80%E4%BD%BF%E7%94%A8%E7%9A%84%E6%AF%94%E7%89%B9%E5%B8%81%E6%8A%80%E6%9C%AF

But would need to link accordingly

Reply:
I update the link too.
Thank you!